package.json

```diff
@@ -18,7 +18,7 @@
     "changeset": "changeset add",
     "lint": "pnpm --filter '*' lint",
     "lint:fix": "pnpm --filter '*' lint:fix",
-    "prepare": "pnpm --filter embroider-css-modules --filter embroider-css-modules-temporary --filter sample-v2-addon build",
+    "prepare": "pnpm --filter './packages/**' --filter sample-v2-addon build",
```

Comment:
The syntax means, the packages in the following locations will be built after `pnpm install`:

- `docs/sample-v2-addon`
- `packages/ember-codemod-remove-ember-css-modules`
- `packages/embroider-css-modules`
- `packages/embroider-css-modules-temporary`
- `packages/type-css-modules`